modules/ROOT/pages/ts-deploy-initialize-fail-onprem.adoc

```diff
@@ -0,0 +1,21 @@
+= Deployment Initialization failed (with On Premise Deployment target)
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Causes
+
+* You do not have the appropriate server group configured or have the wrong name for the server group in your environment for the deployed message flow.
+
+* Your runtime servers are either not set up or are not active.
+
+== Solution
+
+. Log in to Anypoint Platform > *Runtime Manager* in the same environment.
+
+. Click *Servers* and verify that you have the server group `B2BServerGroup` configured and is in `Running` Status.
+
+. If the server group is in a `Stopped` status contact your MuleSoft platform operations or administration team to review and restart the server.
```

Comment:
The next two steps are actually bullet-list options for rectifying the failure of the container step, and they should appear in the same order as in the container step:
```suggestion
* If `B2BServerGroup` is not set up, refer to https://docs.mulesoft.com/partner-manager/2.0/setup#setup[Set Up Anypoint Partner Manager].
* If the server group shows `Stopped` status, contact your MuleSoft platform operations or administration team to review and restart the server.
```